mdx-models/src/main/java/com/mx/models/id/Authentication.java

```diff
@@ -14,6 +14,7 @@ public class Authentication extends MdxBase<Authentication> {
   @Deprecated
   private Boolean canUseRemoteDeposit;
   private List<MfaChallenge> challenges;
+  private String clientDeviceToken;
```

Comment:
@mattnichols do we want to slowly phase out the `iovationToken` and move everything to this new variable?

Reply:
I talked to Joe about this and it seemed like we were going to keep iovationToken. We came to agreement to just add this token so that it was clear it was being created by the third party API

Reply:
Yeah. I'm not sure that genericizing these fields is a good idea. It is possible that more than one token needs to be stored and transmitted.